File: Molinos.Scato.WebPuerto/src/app/shared/modelos/calidad/horarios-exportador.ts
import { Destino } from "@ScatoModels/destino";
import { Exportador } from "@ScatoModels/exportador";
import { MaterialPuerto } from "@ScatoModels/material-puerto";

export interface HorariosExportador {
    id: number;
    moduloDeCarga_Id: number;
    inicio?: Date;
    fin?: Date;
    materialPuerto: MaterialPuerto;
    exportador: Exportador;
    cantidad: number;
    tiempo: string;
  }

File: Molinos.Scato.Dominio/Entidades/HorariosExportador.cs
using System;

namespace Molinos.Scato.Dominio.Entidades
{
    public class HorariosExportador : IIdentificable
    {
        public virtual int Id { get; set; }
        public int ModuloDeCarga_Id { get; set; }
        public virtual DateTime? Inicio { get; set; }
        public virtual DateTime? Fin { get; set; }
        public virtual MaterialPuerto MaterialPuerto { get; set; }
        public virtual Exportador Exportador { get; set; }
        public virtual PlanoDeCargaBodegaDestino PlanoDeCargaBodegaDestino { get; set; }
    }
}